modules/gapi/include/opencv2/gapi/core.hpp

```diff
@@ -1857,7 +1857,7 @@ kmeans(const GArray<Point2f>& data, const int K, const GArray<int>& bestLabels,
 /** @overload
 @note Function textual ID is "org.opencv.core.kmeans3D"
  */
-GAPI_EXPORTS std::tuple<GOpaque<double>,GArray<int>,GArray<Point3f>>
+GAPI_EXPORTS_W std::tuple<GOpaque<double>,GArray<int>,GArray<Point3f>>
```

Comment:
Make sense to add test for this one because it touches a lot of existing functionality.

Reply:
kmeans or what?

Reply:
Yes, add test for `kmeans` because it returns `Point3f` which wasn't supported before

Reply:
Added